cmd/kubeless/deploy.go

```diff
@@ -83,13 +84,21 @@ var deployCmd = &cobra.Command{
 			logrus.Fatal(err)
 		}
 
+		mem, err := cmd.Flags().GetString("memory")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+		if mem != "" {
+			validateMemory(mem)
+		}
+
 		funcType := "PubSub"
 		if triggerHTTP {
 			funcType = "HTTP"
 			topic = ""
 		}
 
-		err = utils.CreateK8sCustomResource(runtime, handler, file, funcName, funcType, topic, ns, deps, description, labels, envs)
+		err = utils.CreateK8sCustomResource(runtime, handler, file, funcName, funcType, topic, ns, deps, description, mem, labels, envs)
```

Comment:
Right. I just extended from the previous code. Will send only the funcSpec as argument